code/modules/antagonists/hivemind/spells/hivemind_loyalty.dm

```diff
@@ -0,0 +1,27 @@
+/obj/effect/proc_holder/spell/self/hive_loyal
+	name = "Concentrated Infiltration"
+	desc = "We prepare for a focused attack on a mind, penetrating mindshield technology, the mindshield will still be present after the attack (toggle)."
+	panel = "Hivemind Abilities"
+	charge_max = 600
```

Comment:
Is there a point to having this on cooldown if its a toggle for the regular spell and doesn't go away?